src/test/java/io/github/enkarin/bookcrossing/service/UserServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package io.github.enkarin.bookcrossing.service;
+
+import io.github.enkarin.bookcrossing.base.BookCrossingBaseTests;
+import io.github.enkarin.bookcrossing.exception.*;
+import io.github.enkarin.bookcrossing.support.TestDataProvider;
+import io.github.enkarin.bookcrossing.user.dto.UserDto;
+import io.github.enkarin.bookcrossing.user.dto.UserProfileDto;
+import io.github.enkarin.bookcrossing.user.dto.UserPublicProfileDto;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+public class UserServiceTest extends BookCrossingBaseTests {
+    @Test
+    void saveUserCorrectUserTest() {
+        final UserDto user = userService.saveUser(TestDataProvider.buildAlex());
+        usersId.add(user.getUserId());
```

Comment:
Try to add method like `createUser(...) `and move this logic `usersId.add(user.getUserId());` there

Reply:
Now let's move createUser to the base class